examples/tv-app/tv-common/include/CHIPProjectAppConfig.h

```diff
@@ -51,5 +51,10 @@
 // overrides CHIP_DEVICE_CONFIG_DYNAMIC_ENDPOINT_COUNT in CHIPProjectConfig
 #define CHIP_DEVICE_CONFIG_DYNAMIC_ENDPOINT_COUNT 16
 
+// For casting, we need to allow more ACL entries, and more complex entries
```

Comment:
OK but note this will expand storage usage, and RAM usage. Space is reserved for every entry, and all those targets and subjects in each one. You may wish to lower the subjects/targets limit to something like 8-10, keep the entries limit sufficiently high, and use more than one entry. There's no right answer here, it's mostly a question of balance. It's implemented this way for the smallest devices, ideally if we have more powerful devices, we can provide a dynamic implementation someday.

Reply:
This is for a tv where long term we should have a dynamic implementation. But for now I just wanted to make sure we don't run out of space during testing.

Reply:
Understood.